dbms/src/Common/Exception.h

```diff
@@ -18,6 +18,7 @@
 #include <Common/StackTrace.h>
 #include <Poco/Exception.h>
 #include <common/defines.h>
+#include <common/logger_useful.h>
```

Comment:
why add this?

Reply:
L313 used `LogFmtDetails`, if some file doesn't inlcude logger_useful.h, try to use `RUNTIME_ASSERT` will cause compilation fail.